package.json

```diff
@@ -41,7 +41,7 @@
     "test-api": "mocha -R dot --recursive tests/api",
     "test-api-log": "ENABLE_API_ACTIVITY_LOGS=true mocha -R dot --recursive tests/api",
     "test-disablehttp": "REQUIRE_HTTPS=true mocha -R dot --recursive tests/disableHttp",
-    "test-enforced": "REQUIRE_ACCESS_TOKEN=true mocha -R dot --recursive tests/enforceToken",
+    "test-enforced": "REQUIRE_ACCESS_TOKEN=true mocha -R dot --recursive tests/enforceToken tests/api/v1/profiles",
```

Comment:
That's correct, all tests under v1/profiles are being called once with REQUIRE_ACCESS_TOKEN = true and once with false @iamigo